apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/CarrierClaimTransformerTest.java

```diff
@@ -50,26 +88,14 @@ public void transformSampleARecord() throws FHIRException, IOException {
             .get();
 
     claim.setLastUpdated(Instant.now());
-    ExplanationOfBenefit eobWithLastUpdated =
-        CarrierClaimTransformer.transform(
-            new TransformerContext(
-                new MetricRegistry(),
-                Optional.of(true),
-                FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting(),
-                new NPIOrgLookup()),
-            claim);
-    assertMatches(claim, eobWithLastUpdated, Optional.of(true));
+    ExplanationOfBenefit eobWithLastUpdated = transformerInterface.transform(claim, true);
+
+    assertMatches(claim, eobWithLastUpdated, Boolean.TRUE);
```

Comment:
These can also be converted to primitives

Reply:
done.

Reply:
Missed this one spot though!